### DISCUSSION TITLE
[Accessibility-Reporting] Why Accessibility-Reporting link can't be part of Accessibility-Metadata?

### DISCUSSION THREAD

**Discussion by abhina-kumar** [w3c/personalization-semantics#258] [Ideas]
Why Accessibility-Reporting link can't be part of Accessibility-Metadata?

**Comment by lwolberg** [comment]
Reporting API discussion is deferred for now.

We clarified in the TF that Reporting API and Well Known Destination are quite different.

**Comment by lwolberg** [comment]
See Discussion #259
